spec/params/keyword_params_spec.rb

```diff
@@ -0,0 +1,123 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe KeywordParams do
+  describe '#initialize' do
+    context('when init with adwords_url_contains') do
+      it('has filter with only adwords_url_contains') do
```

Comment:
No need `(` 😸 

```suggestion
    context 'when init with adwords_url_contains' do
      it 'has filter with only adwords_url_contains' do
```

Reply:
Fixed in https://github.com/nimblehq/ic-rails-phong-bliss/pull/78/commits/e91fb7286fde0e64ca3f24e6374149c1212aeebd